app/assets/stylesheets/modules/global-alert.scss

```diff
@@ -1,7 +1,16 @@
- $global-alert-color: rgb(204, 0, 0);
+$global-alert-color: rgb(204, 0, 0);
+$su-bg-digital-green: rgb(0 133 102);
```

Comment:
This doesn't seem to match the brand guide: https://identity.stanford.edu/design-elements/color/web/  Digital green should be `rgb(0 155 118)`, right?